src/main/java/net/minecraftforge/fluids/FluidAttributes.java

```diff
@@ -141,9 +132,20 @@
      */
     private final int color;
 
+    /**
+     * Amount of time a bucket of the fluid will last for when used as a fuel.
+     *
+     * Note that this will include both things that are hot and things that are actively combusting.
+     * To differentiate them, place them in separate tags, such as forge:fluids/combustible
+     *
+     * For reference, smelting one item takes 200 ticks, one piece of coal = 1600 ticks,
+     * and a bucket of lava lasts for 20000 ticks.
+     */
+    protected int fuelTime = 0;
```

Comment:
javadocs are helpful on PUBLIC methods, so add this information to the getter as well.

Reply:
Removed the fuelTime stuff instead.